### ISSUE TITLE
Get the difference of days between two dates in swift

### ISSUE THREAD

**Issue by myawesomehub** [myawesomehub/Extended#10]
## Overview

- Write an extension to `Int`, and get the difference of days in number by passing two `Date`.
- Make sure to create the extension as `public` and method as `static`.

## Sample

```swift
let twelthApr: Date = Date(timeIntervalSince1970: 1649742445)
let twentyApr: Date = Date(timeIntervalSince1970: 1650433645)

let differenceOfDay: Int = .diffOfDay(from: twelthApr, to: twentyApr) // 8

```

## Code Path

- Sources -> Extended -> Public -> Extension -> Int.swift

## Discussion

- You can discuss about it below in comment.
- You can mail me at mdcode2021@gmail.com

**Comment by JaxTurboNerd** [comment]
Do you also want a method that returns the difference in days to be inclusive of the start date?  In your above example the difference in days = 9.

**Comment by myawesomehub** [comment]
@JaxTurboNerd any of them will work

**Comment by myawesomehub** [comment]
Resolved in #17
